### DISCUSSION TITLE
Some NAS Files Not In Alpha-Numerical Order

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1135] [Fixed Issues]
I find rAudio is changing the order of some files when an NAS folder is added to the playlist. The files are m4a, and are shown correctly by the file manager. The file names in the folder are the same, plus a number. Permissions and ownership are the same for each file.

How does rAudio decide on the track order?

**Comment by rern** [comment]
Library: **rAudio** does the sorting, with A/An/The stripped, except track list of each album.
Playlist: Add a directory/folder - `mpc ls 'NAS/path/to/directory' | mpc add`

**Comment by R101G** [comment]
Thanks @rern , the track metadata was missing.
